core/kernel/tee_ta_manager.c

```diff
@@ -112,15 +112,13 @@ static struct tee_ta_ctx *ts_to_ta_ctx(struct ts_ctx *ctx)
 	panic("bad context");
 }
 
-static bool tee_ta_try_set_busy(struct tee_ta_ctx *ctx)
+static bool tee_ta_try_set_busy_nolock(struct tee_ta_ctx *ctx)
```

Comment:
Is necessary lock ```ctx->flags & TA_FLAG_CONCURRENT``` by mutex_lcok? 

line 119 to 120 may be redundant.

Reply:
good catch. Will refine.